### PR TITLE
chore:fix contactbanner styling

### PR DESCRIPTION
**Pull request by momentiris**
This breaks the contactbanner into a contactblock component instead of having conditionals.
Also aligns spacing with figma designs in the contactblock styling.

### CODE REVIEW

File: sapper/src/routes/case/[slug].svelte [hunk 1]
```diff
@@ -52,7 +52,17 @@
   </div>
 </Layout.Page>
 
-<ContactBanner
+<ContactBlock
   title="Har du en liknande utmaning?"
   text="Vi är redo att hjälpa era att ta det första steget mot att förverkliga er digitala vision. Kontakta oss så berättar vi mer."
-  context="sale" />
+  context="sale">
```
Comment (izabellalarsson) [inline sapper/src/routes/case/[slug].svelte:58]: Remove unused prop

**Review by izabellalarsson — COMMENTED** [review]
Looks good! Think this will result with more copy/paste when including the component, but that might just be me.

File: sapper/src/routes/case/[slug].svelte (hunk 1 above)
Comment (momentiris) [reply sapper/src/routes/case/[slug].svelte]: 👍

File: sapper/src/routes/index.svelte [hunk 2]
```diff
@@ -167,4 +167,4 @@
   </div>
 </Animation.WithScrollFadeIn>
 
-<ContactBanner />
+<ContactBlock />
```
Comment (izabellalarsson) [inline sapper/src/routes/index.svelte:170]: Include the ConnectSymbol

Comment (momentiris) [reply sapper/src/routes/index.svelte:170]: 👍

**Comment by momentiris** [comment]
I think it is cleaner to have one component for everything that is shared between the different types of contact blocks. Right now we have two, but we might have more in the future. And since the parts that are different are totally different, I don't think you really copy/paste that much. Does that make sense?

**Comment by momentiris** [comment]
I see how you will need to copy some stuff on the places where it looks exactly the same. Maybe its better to have two different components then?

**Comment by izabellalarsson** [comment]
Think it might be better, or if the basic one could be called when just using `<ContactBlock/>`

**Comment by momentiris** [comment]
@izabellalarsson What do you think about this?

**Review by izabellalarsson — APPROVED** [review]
Much cleaner! 👍

**Pull request merged**